app/Busline.php

```diff
@@ -7,12 +7,13 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\HasMany;
 use Illuminate\Support\Carbon as Carbon;
+use Illuminate\Support\Collection;
 
 /**
  * Busline model.
  *
- * @property int $id
- * @property string number
+ * @property int               $id
```

Comment:
Why are you putting these whitespaces here?

Reply:
It’s auto formatted due to the ruleset of our choice via php-cs-fixer. It sets variables in one column in phpdoc and is consistent with what we were doing so far.

Reply:
Okaaay. For the record: I hate it.